multidict/__init__.pyi

```diff
@@ -1,12 +1,19 @@
 import abc
-from collections.abc import Iterable, Iterator, Mapping, MutableMapping
-from typing import Generic, TypeAlias, TypeVar, overload
+from typing import (
+    Generic,
+    Iterable,
+    Iterator,
+    Mapping,
+    MutableMapping,
+    TypeVar,
+    overload,
+)
 
 class istr(str): ...
 
 upstr = istr
 
-_S: TypeAlias = str | istr
```

Comment:
TypeAlias doesn't really do anything, so I'd be inclined to leave it out until supported in all our Python versions. I expect pyupgrade will do this automatically anyway.

Again, I only made these changes in the first place, because flake8-pyi highlighted them and suggested that it was safe for all Python versions because it is a stub file and not read at runtime. We don't do this in other projects with inline typing.

Reply:
Okay, resolving this thread.